src/Controls/src/Core/HandlerImpl/ContentPage.Impl.cs

```diff
@@ -80,5 +82,29 @@ protected override void ArrangeOverride(Rectangle bounds)
 		}
 
 
+		#region HotReload
+
+		IView IReplaceableView.ReplacedView => HotReload.MauiHotReloadHelper.GetReplacedView(this) ?? this;
+
+		HotReload.IReloadHandler HotReload.IHotReloadableView.ReloadHandler { get; set; }
+
+		void HotReload.IHotReloadableView.TransferState(IView newView)
+		{
+			//TODO: LEt you hot reload the the ViewModel
+			if (newView is View v)
+				v.BindingContext = BindingContext;
```

Comment:
we probably have to set the parent, and have this virtual

Reply:
Yes, this has a todo: there is more we should do here. But this is a good start!